pyouroboros/dockerclient.py

```diff
@@ -237,6 +237,42 @@ def update(self):
 
                 created = self.client.api.create_container(**new_config)
                 new_container = self.client.containers.get(created.get("Id"))
+
+                # disconnect new container from old networks (with possible
+                # broken config)
+                for network in new_container.attrs['NetworkSettings']['Networks']:
+                    self.client.api.disconnect_container_from_network(
+                        container=new_container.attrs['Id'],
+                        net_id=network,
+                        force=True
+                    )
+
+                # connect the new container to all networks of the old container
+                for network in container.attrs['NetworkSettings']['Networks']:
+                    host_net = self.client.networks.get(network_id=network)
+                    if ('Options' in host_net.attrs
+                        and 'com.docker.network.bridge.default_bridge' in host_net.attrs['Options']
+                        and bool(host_net.attrs['Options']['com.docker.network.bridge.default_bridge'])
```

Comment:
what are the possible values before it is passed to bool(). the bool() operator may not be needed

Reply:
This was indeed an error, I misinterpreted the behaviour of `bool()`.  
The possible values are `true` and `false`, both as `String`.